src/components/dropdowns/Dropdown.jsx

```diff
@@ -14,13 +14,23 @@ type PropsType = $ReadOnly<{
   name: string,
   links: Array<LinksType>,
   initiallyOpened?: boolean,
+  colorMode?: 'dark' | 'light',
```

Comment:
I'd keep what we have for selects `color?: 'default' | 'white'`

Reply:
sure, fixed - I guess in this case it makes sense to keep it consistent with Select component, however I'm not a huge fan of naming things with color names as if in the future we change the color to e.g. light gray, giving it prop color="white' could be slightly confusing :) @bartoszlorek